widget/playground/src/components/Slider/Slider.styles.ts

```diff
@@ -30,19 +30,25 @@ export const RangeWrapper = styled('div', {
   },
   '& .range-custom': {
     height: '3px',
-    backgroundColor: '$secondary500',
+    backgroundColor: '$neutral300',
     '-webkit-appearance': 'none',
+    outline: 'none',
+    appearance: 'none',
+
     '&::-webkit-slider-thumb': {
       '-webkit-appearance': 'none',
+      appearance: 'none',
       width: '10px',
       height: '8px',
```

Comment:
I saw a small border-radius for webkit-slider-thumb in Figma. If it's right please add it.

Reply:
fixed